### PR TITLE
fix(css/prefixer): generate at-rules based on env

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Don't generate unnecessary `at-rules` based on `env`

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_prefixer/src/prefixer.rs [hunk 1]
```diff
@@ -553,6 +553,16 @@ struct Prefixer {
     added_declarations: Vec<Declaration>,
 }
 
+impl Prefixer {
+    fn add_at_rule(&mut self, prefix: Prefix, at_rule: AtRule) {
```
Comment (kdy1) [inline crates/swc_css_prefixer/src/prefixer.rs:557]: I think `at_rule` should be `&AtRule`

File: crates/swc_css_prefixer/src/prefixer.rs [hunk 2]
```diff
@@ -594,85 +604,87 @@ impl VisitMut for Prefixer {
...
+                        Prefix::Ms,
+                        AtRule {
+                            span: DUMMY_SP,
+                            name: AtRuleName::Ident(Ident {
+                                span: DUMMY_SP,
+                                value: "-ms-viewport".into(),
+                                raw: None,
+                            }),
+                            prelude: n.prelude.clone(),
+                            block: original_simple_block.clone(),
```
Comment (kdy1) [inline crates/swc_css_prefixer/src/prefixer.rs:618]: So we clone only if required

**Review by kdy1 — CHANGES_REQUESTED** [review]
Thanks!

**Comment by alexander-akait** [comment]
@kdy1 Fixed :+1:

**Review by kdy1 — APPROVED** [review]
Thanks!

---
swc-bump:
 - swc_css_prefixer

**Pull request merged**